### ISSUE TITLE
add functionality to update user provided unit

### ISSUE THREAD

**Issue by fdschneider** [EcologicalTraitData/traitdataform#3]
After running the standardize.taxonomy function, the user provided unit might need to be changed. a re-running of as.traitdata is invalid. Also the mapping function does not provide this. It might not be necessary to write an own function, but providing a one-liner that maps units according to column "traitName".

**Comment by fdschneider** [comment]
```r
dataset1std$traitUnit <- c("mm", "unitless", rep("", 7))[as.integer(dataset1std$traitName)]
```

**Comment by fdschneider** [comment]
would be solved by `merge()` method, see #7
